debezium-core/src/main/java/io/debezium/util/NumberConstants.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.util;
+
+
+import java.math.BigDecimal;
+import java.nio.ByteBuffer;
+import java.util.Date;
+
+/**
+ * A set of Numbers.
+ *
+ * @author pmx
+ */
+public class NumberConstants {
```

Comment:
Could you add your real name? Makes it easier to grasp the history and work with this code down the road :)